### ISSUE TITLE
Bitrise envman fastlane 2.68.0 issue 

### ISSUE THREAD

**Issue by antwllm** [fastlane/fastlane#11099]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Hello !

It seems that the version of fastlane 2.68.0 has some issues with envman (Bitrise tool). Since the update, the execution of my lane stops at the line I use envman.

Nothing happens, no error, just stop.

The same Fastfile is working perfectly on the version 2.67.0

**Comment by milch** [comment]
Hi @antwllm! Can you run the lane that is failing with `fastlane --capture_output <lane>` and paste the output here please?

**Comment by antwllm** [comment]
Please find bellow the logs from Bitrise

`envman` is a tool provided by Bitrise

Fastlane version:
$ fastlane "--version"
fastlane installation at path:
/usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.68.0/bin/fastlane
-----------------------------
fastlane 2.68.0
Run Fastlane
$ fastlane "bitrise"
[13:08:08]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[13:08:10]: -------------------------------------------------
[13:08:10]: --- Step: Verifying required fastlane version ---
[13:08:10]: -------------------------------------------------
[13:08:10]: Your fastlane version 2.68.0 matches the minimum requirement of 2.50.0  ✅
[13:08:10]: ------------------------------
[13:08:10]: --- Step: default_platform ---
[13:08:10]: ------------------------------
[13:08:10]: -----------------------------
[13:08:10]: --- Step: import_from_git ---
[13:08:10]: -----------------------------
[13:08:10]: Cloning remote git repo...
[13:08:10]: $ GIT_TERMINAL_PROMPT=0 git clone 'git@appsolute-git.fr:fastalane/fastlane.git' '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171130-1994-1mavogx/fastlane.git' --depth 1 -n 
[13:08:10]: ▸ Cloning into '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171130-1994-1mavogx/fastlane.git'...
[13:08:11]: ▸ Warning: Permanently added 'appsolute-git.fr,213.32.105.118' (ECDSA) to the list of known hosts.
[13:08:12]: $ cd '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171130-1994-1mavogx/fastlane.git' && git checkout HEAD 'import-files/appsolute-base-import-v3'
[13:08:12]: $ cd '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171130-1994-1mavogx/fastlane.git' && git checkout HEAD 'import-files/actions'
[13:08:12]: ▸ error: pathspec 'import-files/actions' did not match any file(s) known to git.
[13:08:12]: Driving the lane 'ios bitrise' 🚀
[13:08:12]: --------------------------------------------------------------------
[13:08:12]: Step: envman add --key FASTLANE_SCHEME --value ${FASTLANE_SCHEME}
[13:08:12]: --------------------------------------------------------------------
[13:08:12]: $ envman add --key FASTLANE_SCHEME --value ${FASTLANE_SCHEME}

**Comment by milch** [comment]
With the information you've given I can not reproduce this issue in my setup. I've installed the latest version of envman from https://github.com/bitrise-io/envman (1.1.8)

**Comment by antwllm** [comment]
Hi @milch,

I switched back to 2.67.0 and it is working but on 2.68.0.

So I really don't know where it is coming from.

**Comment by ohayon** [comment]
Sounds like youre back up and running @antwllm?! I am going to close this issue for now, let us know if you are still having issues. Thanks! 🚀

**Comment by antwllm** [comment]
Hello !

I have tried the release 2.68.1 and I still have the same issue.

It doesn't happen on the 2.67.0

```
Total time: 7.496812ms
|                                                                              |
+---+---------------------------------------------------------------+----------+
| ✓ | cache-push@2.0.2                                              | 3.40 sec |
+---+---------------------------------------------------------------+----------+
| Update available: 2.0.2 -> 2.0.3                                             |
+---+---------------------------------------------------------------+----------+
                                          ▼
+------------------------------------------------------------------------------+

| (7) script@1.1.5                                                             |
+------------------------------------------------------------------------------+
| id: script                                                                   |
| version: 1.1.5                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: bash                                                                |
| time: 2017-12-04T23:56:27-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
+ gem install fastlane --version 2.68.1 --no-document
Successfully installed tty-screen-0.6.3
Successfully installed fastlane-2.68.1
2 gems installed
|                                                                              |
+---+---------------------------------------------------------------+----------+
| ✓ | script@1.1.5                                                  | 6.71 sec |
+---+---------------------------------------------------------------+----------+
                                          ▼
Switching to workflow: primary
Switching to workflow: build-dev
+------------------------------------------------------------------------------+

| (0) fastlane@2.3.9                                                           |
+------------------------------------------------------------------------------+
| id: fastlane                                                                 |
| version: 2.3.9                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2017-12-04T23:56:34-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[23:56:34]  * [OK] Step dependency (go) installed, available. 
Configs:
- WorkDir: /Users/vagrant/git
- Lane: bitrise
- UpdateFastlane: false
Expand WorkDir
Expanded WorkDir: /Users/vagrant/git
Determine desired Fastlane version
Checking Gemfile.lock (/Users/vagrant/git/Gemfile.lock) for fastlane gem
Gemfile.lock does not exist
Using system installed Fastlane
Fastlane version:
$ fastlane "--version"
fastlane installation at path:
/usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.68.1/bin/fastlane
-----------------------------
fastlane 2.68.1
Run Fastlane
$ fastlane "bitrise"
[23:56:39]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[23:56:41]: -------------------------------------------------
[23:56:41]: --- Step: Verifying required fastlane version ---
[23:56:41]: -------------------------------------------------
[23:56:41]: Your fastlane version 2.68.1 matches the minimum requirement of 2.50.0  ✅
[23:56:41]: ------------------------------
[23:56:41]: --- Step: default_platform ---
[23:56:41]: ------------------------------
[23:56:41]: -----------------------------
[23:56:41]: --- Step: import_from_git ---
[23:56:41]: -----------------------------
[23:56:41]: Cloning remote git repo...
[23:56:41]: $ GIT_TERMINAL_PROMPT=0 git clone 'git@appsolute-git.fr:fastalane/fastlane.git' '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171204-2146-ienfa4/fastlane.git' --depth 1 -n 
[23:56:41]: ▸ Cloning into '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171204-2146-ienfa4/fastlane.git'...
[23:56:42]: ▸ Warning: Permanently added 'appsolute-git.fr,213.32.105.118' (ECDSA) to the list of known hosts.
[23:56:43]: $ cd '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171204-2146-ienfa4/fastlane.git' && git checkout HEAD 'import-files/appsolute-base-import-v3'
[23:56:43]: $ cd '/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/fl_clone20171204-2146-ienfa4/fastlane.git' && git checkout HEAD 'import-files/actions'
[23:56:43]: ▸ error: pathspec 'import-files/actions' did not match any file(s) known to git.
[23:56:43]: Driving the lane 'ios bitrise' 🚀
[23:56:43]: --------------------------------------------------------------------
[23:56:43]: Step: envman add --key FASTLANE_SCHEME --value ${FASTLANE_SCHEME}
[23:56:43]: --------------------------------------------------------------------
[23:56:43]: $ envman add --key FASTLANE_SCHEME --value ${FASTLANE_SCHEME}
```
